src/sonic-host-service/org.sonic.hostservice.conf

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE busconfig PUBLIC
+ "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<busconfig>
+
+  <!-- Only root can own the bus -->
+
+  <policy user="root">
+    <allow own_prefix="org.SONiC.HostService"/>
+  </policy>
+
+  <!-- Allow user "root" to invoke methods on the bus -->
```

Comment:
What is the policy shall I add if I want to make rest server run on non root user?

Reply:
This file does not control the user as which the REST server runs, only the users/groups who can access the bus. Therefore, if and when you change REST server to run as a non-root user; say username `rest`, group `mgmt`; then you need to change the policy to allow either `user="rest"` or `group="mgmt"`.

Reply:
Fine, thanks